src/doc/src/reference/external-tools.md

```diff
@@ -265,6 +265,12 @@ cargo `(?<command>[^ ]+)` into an invocation of an external tool
 `cargo-${command}`. The external tool must be present in one of the user's
 `$PATH` directories.
 
+> Note: Cargo looks for custom subcommands in `$PATH `and then `$CARGO_HOME/bin`, 
```

Comment:
This seems a bit incorrect. I may emphasize two things:

* By default, tools in `$CARGO_HOME/bin` take precedence over tools in `$PATH`.
* Users can put `$CARGO_HOME/bin` in `$PATH` to control the precedence.

Reply:
> By default, tools in $CARGO_HOME/bin take precedence over tools in $PATH.

Yes, Indeed.

Reply:
I've polished the description to the best of my ability, hoping to get it right.